sky/data/storage_utils.py

```diff
@@ -223,7 +219,16 @@ def get_excluded_files_from_gitignore(src_dir_path: str) -> List[str]:
 def get_excluded_files(src_dir_path: str) -> List[str]:
     # TODO: this could return a huge list of files,
     # should think of ways to optimize.
-    """List files and directories to be excluded."""
+    """List files and directories to be excluded.
+
+    Args:
+        src_dir_path (str): The path to the source directory.
+
+    Returns:
+        A list of relative paths to files and directories to be excluded from
+        the source directory.
+    """
+    assert os.path.isdir(src_dir_path), src_dir_path
```

Comment:
Need to call `os.path.expanduser` here. Actually I'd rather just remove this because we already have a check below for `isfile` - or we could change that check to do `not os.path.isdir(...)`

Reply:
Good catch! Removed the test here and added an additional check below in the if block